test/components/userComments.test.js

```diff
@@ -2,10 +2,25 @@ import UserComments from '../../components/UserComments'
 
 describe('<UserComments  />', () => {
   it('it renders correctly', () => {
+    const comments = [{
+      id: 'ID',
```

Comment:
`id`, `articleId` aren't needed. But `createdAt` is

Reply:
I can remove articleId but if I remove id, the test fails

Reply:
What's the error message? Are you using the ID for the key? If so, probably best to add it back to the query then.

Reply:
Warning: Each child in an array or iterator should have a unique "key" prop.%s%s See https://fb.me/react-warning-keys for more information.%s